src/NServiceBus.Core/Transports/Learning/LearningTransportMessagePump.cs

```diff
@@ -24,7 +24,10 @@ public Task Init(Func<MessageContext, Task> onMessage, Func<ErrorContext, Task<E
             transactionMode = settings.RequiredTransactionMode;
 
             path = Path.Combine(basePath, settings.InputQueue);
-            
+            Directory.CreateDirectory(Path.Combine(path, ".committed"));
```

Comment:
One last question, why has this line to create the committed dir been added here? I'm not seeing anywhere else in this PR that seems to need it?